bootstrap.sh

```diff
@@ -31,10 +31,13 @@ do
   cd  ${service}
   git filter-repo --to-subdirectory-filter ${service}
 
-  # Search for <parent> tag and replace the next three lines -- groupId, artifcatId, and version
-  # Doesn't modify the pom.xml for each module's bom package (that still points to shared-config ... for now)
+  # Search for <parent> tag in module pom and replace the next three lines -- groupId, artifcatId, and version
   sed -i -e '/<parent>/{N;s/com.google.cloud/com.google.api/;N;s/google-cloud-shared-config/google-cloud-java/;N;s/<version>.*<\/version>/<version>0.0.1-SNAPSHOT<\/version>/}' ${service}/pom.xml
 
+  NAME=$(jq -r '.distribution_name' ${service}/.repo-metadata.json | cut -d ':' -f 2)
+  # Search for <parent> tag in module bom and replace the next three lines -- groupId, artifcatId, and version
+  sed -i -e '/<parent>/{N;s/com.google.cloud/com.google.api/;N;s/google-cloud-shared-config/google-cloud-java/;N;s/<version>.*<\/version>/<version>0.0.1-SNAPSHOT<\/version>\n    <relativePath>..\/..\/pom.xml<\/relativePath>/}' ${service}/${NAME}-bom/pom.xml
```

Comment:
Adding in <relativePath>...</relativePath> for the bom to find the google-cloud-java artifact. Default behavior it it only searches up one directory (needs two).